includes/apple-exporter/components/class-table.php

```diff
@@ -53,30 +60,66 @@ public function register_specs() {
 		// Get information about the currently loaded theme.
 		$theme = \Apple_Exporter\Theme::get_used();
 
+		// Get Dark Table Colors.
+		$table_border_color_dark            = $theme->get_value( 'table_border_color_dark' );
+		$table_body_background_color_dark   = $theme->get_value( 'table_body_background_color_dark' );
+		$table_body_color_dark              = $theme->get_value( 'table_body_color_dark' );
+		$table_header_background_color_dark = $theme->get_value( 'table_header_background_color_dark' );
+		$table_header_color_dark            = $theme->get_value( 'table_header_color_dark' );
+
+		// If all dark table styles are empty, do not add conditional styles.
+		$this->dark_table_colors_exist =
+			! empty( $table_border_color_dark ) ||
+			! empty( $table_body_background_color_dark ) ||
+			! empty( $table_body_color_dark ) ||
+			! empty( $table_header_background_color_dark ) ||
+			! empty( $table_header_color_dark );
+
+		$dark_table_conditional = [];
+		if ( $this->dark_table_colors_exist ) {
+			$dark_table_conditional = [
+				'conditional' => [
+					[
+						'style'      => 'dark-table',
+						'conditions' => [
+							'minSpecVersion'       => '1.14',
+							'preferredColorScheme' => 'dark',
+						],
+					],
+				],
+			];
```

Comment:
Component level conditional that decides which `componentStyles` are applied to tables.  Either `default-table` or `dark-table` depending on the `preferredColorScheme`.

The `minSpecVersion` was applied to the old conditionals so I brought it along here.  Could be that AN didn't support dark mode before that version.

Reply:
That's correct, ANF didn't support dark mode before 1.14